### ISSUE TITLE
plugin breaks colored maven output on windows

### ISSUE THREAD

**Issue by jpraet** [fabric8io/docker-maven-plugin#962]
![image](https://user-images.githubusercontent.com/2212615/36836487-bdd9aee2-1d39-11e8-96a1-92824c8f1c28.png)

As you can see in the screenshot. Maven (3.5) colored output is working fine until the docker-maven-plugin executes. For that plugin, and for all remaining plugins in the maven build, there is no more colored output.

**Comment by jpraet** [comment]
There is a static call to Ansi.setEnabled(false) here: https://github.com/fabric8io/docker-maven-plugin/blob/ac02ae40c14bff2728007e8cd135ed441ab2960c/src/main/java/io/fabric8/maven/docker/util/AnsiLogger.java#L208

**Comment by rhuss** [comment]
Yep, before 3.5 ANSI code broke Windows update. Will update it to not disable it on Windows for Maven >= 3.5

thanks for the heads up ! btw, would also integrate a PR, might actually be a bit faster, considering my backlog ;)

**Comment by wrosenuance** [comment]
This should be fixed by the now-merged PR #1199.
